hermes/src/main.rs

```diff
@@ -70,9 +72,20 @@ async fn init() -> Result<()> {
 }
 
 #[tokio::main]
+#[tracing::instrument]
 async fn main() -> Result<()> {
     env_logger::init();
 
+    // Initialize a Tracing Subscriber
+    tracing::subscriber::set_global_default(
```

Comment:
Should be able to compose this with [tracing-opentelemetry](https://crates.io/crates/tracing-opentelemetry) if we want this in DataDog and so on.